common-clusterpreferences-bundle.yaml

```diff
@@ -2,6 +2,10 @@
 apiVersion: instancetype.kubevirt.io/v1alpha2
 kind: VirtualMachineClusterPreference
 metadata:
+  annotations:
+    openshift.io/documentation-url: https://github.com/kubevirt/common-instancetypes
+    openshift.io/provider-display-name: KubeVirt
```

Comment:
Is the an appropriate icon for alpine? Maybe a generic one?

Reply:
As above, I doubt we will ship this downstream so I didn't want to add an icon here.

Reply:
What about OKD? Can't it be used there?